docs/stellar-core_example.cfg

```diff
@@ -481,6 +481,13 @@ RUN_STANDALONE=false
 #     in the header invariant/LiabilitiesMatchOffers.h.
 #     The overhead may cause slower systems to not perform as fast as the rest
 #     of the network, caution is advised when using this.
+# - "EventsAreConsistentWithEntryDiffs"
+#     Setting this will cause additional work on each operation apply - it
+#     checks that all asset movements and events provide equivalent 
+#     balance changes. The invariant required both EMIT_CLASSIC_EVENTS and
+#     BACKFILL_STELLAR_ASSET_EVENTS config flags to be enabled. This is also
+#     a "strict" invariant, which means your node will shutdown if the invariant
+#     triggers.
```

Comment:
thanks for adding this comment for the new invariant with detailed explanation. 
leaving this comment as a reminder that we should have a version of scheduled test (e.g. the full parallel catchup without skipping sig-verification https://github.com/stellar/stellar-supercluster/issues/320) where this is enabled, so we have the events meta checking to be part of the regular CI workflow.